### PR TITLE
Add the parameter "properties" to Trace

### PR DESCRIPTION
**Pull request by siwei-zhang-work**
Add the parameter "properties" introduced by commit 499408e3979cab403a5d58 in the server (incubator).

Link to the PR for the commit: [eclipse-tracecompass-incubator/org.eclipse.tracecompass.incubator#79](https://github.com/eclipse-tracecompass-incubator/org.eclipse.tracecompass.incubator/pull/79)

Part added:
```
        properties:
          type: object
          additionalProperties:
            type: string
            description: The trace's properties
          description: The trace's properties
```

### CODE REVIEW

**Review by PatrickTasse — COMMENTED** [review]
I'm not sure why this happens or if it can be prevented, but the order of properties of all existing elements has been shuffled, making this very hard to review. I think we were able to avoid this before, but I don't remember.

**Comment by bhufmann** [comment]
> I'm not sure why this happens or if it can be prevented, but the order of properties of all existing elements has been shuffled, making this very hard to review. I think we were able to avoid this before, but I don't remember.

Yes, that happens to me too. One factor in how it is generated is the version of swagger. What I've been doing is that I do a manual diff and only copy the new parts or change parts that I know I updated. Maybe we need a commit that generates the existing swagger and then create a commit with the actual changes.

File: API-proposed.yaml
Comment (bhufmann) [inline API-proposed.yaml:1]: I think you removed manually added proposed APIs in this file. These changes should remain.

**Review by bhufmann — CHANGES_REQUESTED** [review]
as per previous comment

**Comment by bhufmann** [comment]
replaced by #107

**Pull request closed without merging**